### DISCUSSION TITLE
How does uci enable Standard Variant Mode

### DISCUSSION THREAD

**Discussion by jialaolian11** [fairy-stockfish/Fairy-Stockfish#677] [Q&A]
Hello, Sir, since I have not tested how the king rook shift of the AI engine moves, I would like to ask you whether the syntax of this uci command is wrong. Is chess960 disabled? The difficulty of enabling the standard variant is 19. setoption name UCI_Chess960 value false standard value true Skill Level value 19

**Comment by ianfab** [comment]
Please start reading some documentation that helps with command line usage of Fairy-SF, such as https://github.com/fairy-stockfish/Fairy-Stockfish/wiki/Command-line, https://www.wbec-ridderkerk.nl/html/UCIProtocol.html and https://fairy-stockfish.github.io/chess-variant-standards/.

Generally if you are not familiar with how to use the UCI protocol I would recommend to use a GUI instead of the command line.

**Reply by jialaolian11** [reply]
Thank you very much for your answer

**Comment by jialaolian11** [comment]
I am sorry that I did not learn the uci syntax, I used this uci command, AI still out of the Chess960 moves, please tell me whether my syntax is wrong, my idea is to open the standard variant of AI
uci
setoption name UCI_Chess960 value false
setoption name Skill Level value -19
setoption name UCI_Variant value standard
position fen rnbkq1n1/p3b3/2p2rp1/1p3p1p/3Pp2N/PBQ1P1P1/1PP1N2P/R1BK1R2 w Kk - 0 15
go depth 1
